src/AlgebraicGeometry/ToricVarieties/Subvarieties/constructors.jl

```diff
@@ -6,12 +6,8 @@
     toric_variety::AbstractNormalToricVariety
     defining_ideal::MPolyIdeal
     function ClosedSubvarietyOfToricVariety(toric_variety::AbstractNormalToricVariety, defining_ideal::MPolyIdeal)
-      if !is_simplicial(toric_variety)
-        throw(ArgumentError("Currently, closed subvarieties are only supported for simplicial toric varieties"))
-      end
-      if parent(gens(defining_ideal)[1]) != cox_ring(toric_variety)
-        throw(ArgumentError("The defining ideal must be contained in the Cox ring of the toric supervariety"))
-      end
+      @req is_simplicial(toric_variety) "Currently, closed subvarieties are only supported for simplicial toric varieties"
+      @req parent(gens(defining_ideal)[1]) == cox_ring(toric_variety) "The defining ideal must be contained in the Cox ring of the toric supervariety"
```

Comment:
BTW I should point out that I did not try to address the question whether to use `==` or `===` to compare parents, base rings etc. -- but while making this PR I noticed that we do both frequently, and there is no clear pattern.

Perhaps that's OK; or perhaps we should agree to e.g. always use `===` and then add this to the developer guide, and announce it to all devs (e.g. via oscar-dev).

Meta: perhaps that's something we should indeed do more often (announcing changes on how we do things on oscar-dev), as mostly nobody is able to follow all PRs and Slack communication etc., so it's easy to miss out on some change. E.g. that we don't use `Markdown.@doc` anymore for docstrings anymore.

Reply:
I would indeed appreciated announcements on such general changes.

Regarding `==` vs. `===`, that is yet another good point...